client/components/Application/Sidebar.tsx

```diff
@@ -1,44 +1,154 @@
+import * as store from '@client/store'
+import { languages } from '@locale/index'
+import TranslateIcon from '@mui/icons-material/Translate'
 import Box from '@mui/material/Box'
-import Browser from './Browser'
-import LowerMenu from './LowerMenu'
-import sidebarLogo from '../../assets/ODE_sidebar_logo.svg'
 import Button from '@mui/material/Button'
-import * as store from '@client/store'
+import FormControl from '@mui/material/FormControl'
+import InputLabel from '@mui/material/InputLabel'
+import MenuItem from '@mui/material/MenuItem'
+import Select from '@mui/material/Select'
+import Stack from '@mui/material/Stack'
+import i18next from 'i18next'
 import { useTranslation } from 'react-i18next'
+import sidebarLogo from '../../assets/ODE_sidebar_logo.svg'
+import reportIssueIcon from '../../assets/report_issue_icon.svg'
+import userGuideIcon from '../../assets/user_guide_icon.svg'
+import Browser from './Browser'
 
 export default function Sidebar() {
-  const { t } = useTranslation()
   return (
-    <Box
-      className="sidebar"
```

Comment:
@roll the reason you are having the e2e test failing is because of this. The test is looking for the className `sidebar` if you add the class, it will work again.

Reply:
Thanks for finding it out!